data/author

```diff
@@ -152,3 +152,5 @@ Portable Jukebox:Will BL
 Lapis Warps:Will BL
 GVault:Will BL
 Gshops:Will BL
+PastelWonderland:Stereo528
+PastelWonderland:HiItsDevin_
```

Comment:
I think you should add Devin to Consistency+ Devs as well. Hes considered a developer in the discord.

Reply:
oh uh.. you dont have to if you dont want to, i barely contribute...

Reply:
do it anyway

Reply:
Done, anyone who has dev role is dev smh.